test/regress/regress1/ho/NUM925^1.p

```diff
@@ -1,4 +1,4 @@
-% COMMAND-LINE:  --uf-ho
+% COMMAND-LINE:  --uf-ho --ho-elim
```

Comment:
Why did you need to add the option?

Reply:
Because we answer unknown otherwise.

Reply:
But we were solving them without the pass before? Why is that?

Reply:
This appears to be random.

Reply:
I see.